### PR TITLE
String calculator part 2 - change error message and add more invalid input examples

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/tdd/string-calculator-part-2/_index.md [hunk 1]
```diff
@@ -102,4 +112,4 @@ add("//[[][[][&&]\n1[2[3&&4")
 
 - All functionality from part 1 should still work.
```
Comment (sheenarbw) [inline content/projects/tdd/string-calculator-part-2/_index.md]: please get out of the habit of putting blank lines between bullet points.

File: content/projects/tdd/string-calculator-part-2/_index.md [hunk 2]
```diff
@@ -72,28 +70,40 @@ If the string does not abide by any of these formats, it should be considered in
 
 ```
 add("//;\n1000;1;2;")
```
Comment (sheenarbw) [inline content/projects/tdd/string-calculator-part-2/_index.md]: This project is generally only done in Python, but these examples are in JS. I think we can write this ins a language-agnostic way. How about this:

```
For example, each of the following function calls should raise/throw an error/exception with the message "invalid input":

- `add("//;\n1000;1;2;")`
- add(...something else)
- etc
```

Comment (sheenarbw) [reply content/projects/tdd/string-calculator-part-2/_index.md]: There is some other weird formatting in this file as well. Might as well clean it up while we are here. 

There is a codeblock of type markdown that contains a list. 

```md
- "integer,integer,integer" e.g "1,2" or "1,2,3,4"
- "integer \n integer,integer e.g "1\n2,3"
- "//delimiter \n integer delimiter integer" e.g "//;\n1;2"
- "//[delimiter][delimiter]\n integer delimiter integer" e.g "//[\*][%]\n1\*2%3"
```

It should be more like:

- `"integer,integer,integer"` e.g `"1,2" or "1,2,3,4"`
- etc

**Review by sheenarbw — CHANGES_REQUESTED** [review]
weird formatting

File: content/projects/tdd/string-calculator-part-2/_index.md [hunk 3]
```diff
@@ -54,52 +52,46 @@ add("//[abc][777][:(]\n1abc27773:(1")
 
 ### 3. Modify the add function so that it can handle invalid input
 
-If the string passed in is invalid, your code should be able to detect this and throw an error.
+If the string passed in is invalid, your code should be able to detect this and throw throw this error/exception `invalid input`.
```
Comment (Samantha-Hampton) [inline content/projects/tdd/string-calculator-part-2/_index.md:55]: you've repeated the word "throw"

**Review by Samantha-Hampton — CHANGES_REQUESTED** [review]
One small change - see comment - and one suggestion.

It's great that there are more examples of invalid input, but perhaps good to say that it is not an exhaustive list of invalid input that could be tested in the project submission - since the main thing is that their code abides by the accepted formats and all else is invalid. 

Let me know if you have any thoughts on this :)

**Pull request closed without merging**